### PR TITLE
Verification CGU

### PR DESCRIPTION
**Pull request by magdignomad**
First verification and correction of the FAQs

### CODE REVIEW

File: qna/DE-fiskaltrust-sandbox.md [hunk 1]
```diff
@@ -11,14 +11,12 @@ How can I test my implementation?
 What is the fiskaltrust sandbox?
 
 ## Metadata tags
-lang-en, market-all, middleware, portal
+lang-en, market-de, middleware
```
Comment (AxelKutschera) [inline qna/DE-fiskaltrust-sandbox.md:14]: Should it be tagged with market-at and market-fr too?

Comment (magdignomad) [reply qna/DE-fiskaltrust-sandbox.md:14]: No! It is with this screenshot and information only valueable for the german market!

File: qna/DE-fiskaltrust-sandbox.md [hunk 2]
```diff
@@ -11,14 +11,12 @@ How can I test my implementation?
...
 
 ## Answer
 
 We offer a test system called sandbox. The sandbox is fully functional just like our live system. Depending on the country, it can be reached via the following URLs:
```
Comment (fiskaltrust-MBI) [inline qna/DE-fiskaltrust-sandbox.md:18]: If we are making it specific to Germany this line should be changed to: "For German market, the sandbox can be reached via the following URL"

File: qna/AT-carefree-extension-after-expired-grace-period.md [hunk 3]
```diff
@@ -4,12 +4,12 @@ How to extend Carefree products which expired beyond the grace period?
 
 ## Metadata tags
 
-lang-en, market-at, bundle-carefree, portal, cashbox management, PosDealer
+lang-en, market-at, bundle-carefree, portal, PosOperator, PosDealer, Consultant
 
 ## Answer
 
-The subscription for digital products offered by fiskaltrust has a validity date, which, upon expiration, enters a 3-months-long grace period. To ensure the continuity of service, the subscription must be extended before the end of the grace period. After the expiration of the grace period it will not be possible to extend the subscription via portal, and a request for assistance will have to be placed with the fiskaltrust Support. 
+The subscription for digital products offered by fiskaltrust has a validity date, which, upon expiration, enters a 3-months-long grace period. To ensure the continuity of service, the subscription must be extended before the end of the grace period. After the expiration of the grace period it will not be possible to extend the subscription via portal, and a request for assistance will have to be placed with the fiskaltrust Support via eMail.
```
Comment (fiskaltrust-MBI) [inline qna/AT-carefree-extension-after-expired-grace-period.md:11]: I don't see the need for specifying that this should go via email. Fiskaltrust has defined support channels and such requests can go via any of them. We are going to change email-based support to a form based one in some near future and then we'd have to update all documents. Let's remove this part.

File: qna/DE-fiskaltrust-sandbox.md (hunk 2 above)
Comment (magdignomad) [reply qna/DE-fiskaltrust-sandbox.md]: changed

File: qna/AT-carefree-extension-after-expired-grace-period.md (hunk 3 above)
Comment (magdignomad) [reply qna/AT-carefree-extension-after-expired-grace-period.md]: changed

**Pull request merged**